src/pages/dar_collection_review/DarVoteReview.js

```diff
@@ -0,0 +1,6 @@
+import DarCollectionReview from "./DarCollectionReview";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9480935)
```suggestion
import DarCollectionReview from './DarCollectionReview';
```